internal/bminventory/inventory.go

```diff
@@ -1642,7 +1643,7 @@ func (b *bareMetalInventory) updateHostsData(ctx context.Context, params install
 			params.ClusterUpdateParams.HostsNames[i].ID, params.ClusterID).Error
 		if err != nil {
 			log.WithError(err).Errorf("failed to find host <%s> in cluster <%s>",
-				params.ClusterUpdateParams.HostsRoles[i].ID, params.ClusterID)
+				params.ClusterUpdateParams.HostsNames[i].ID, params.ClusterID)
```

Comment:
is it just me or is it a bug? if so why how did unit/subsystem tests passed?

Reply:
It's a bug but only the value of the host ID returned in the error string.

Reply:
right